pyoverkiz/obfuscate.py

```diff
@@ -42,7 +42,12 @@ def obfuscate_sensitive_data(data: dict[str, Any]) -> JSON:
         ]:
             data[key] = obfuscate_string(value)
 
-        if value in ["core:NameState", "homekit:SetupCode", "homekit:SetupPayload"]:
+        if value in [
+            "core:NameState",
+            "homekit:SetupCode",
+            "homekit:SetupPayload",
+            "core:SSIDState",
```

Comment:
What's the value looks like?

Reply:
Just the string value of an SSID. But we should filter this out as it can be quite personally. Same as we do for HomeKit codes.